src/server/services/generation/generation.service.ts

```diff
@@ -940,6 +945,7 @@ const getImageGenerationData = async (id: number): Promise<Generation.Data> => {
       m.id "modelId",
       m.name "modelName",
       m.type "modelType",
+      m.minor,
```

Comment:
I don't think we need this one here, can be safely removed
```suggestion
```

Reply:
ahh, good catch